webapp/src/snippetBuilder.tsx

```diff
@@ -0,0 +1,339 @@
+                className: "approve positive"
+            }
+        ];
+
+        const currQ = config.questions[currentQuestion];
+
+        // TODO: Workspace component
+
+        return (
+            <sui.Modal isOpen={visible} className="snippetBuilder" size="large"
+                closeOnEscape={false} closeIcon={false} closeOnDimmerClick={false} closeOnDocumentClick={false}
+                dimmer={true} buttons={actions} header={config.name}
+            >
+                <div>
+                    <div className="list">
+                        {currQ &&
+                            <div>
+                                <div>{currQ.title}</div>
+                                <div className='list horizontal'>
+                                    {currQ.inputs.map((input: any) =>
+                                        <div>
+                                            <sui.Input
+                                                label={input.label && input.label}
+                                                value={answers[input.answerToken] || ''}
+                                                onChange={this.textInputOnChange(input.answerToken)}
+                                            />
+                                        </div>
+                                    )}
+                                </div>
+                            </div>
+                        }
+                    </div>
+                    <div id="snippetBuilderOutput">
+                        {projectView && <md.MarkedContent markdown={this.generateOutputMarkdown(output)} parent={projectView} />}
+                    </div>
+                </div>
+            </sui.Modal>
+        )
+    }
+}
+
+// This will be passed down as a prop
+const staticConfig: any = {
+    name: "Sprite Builder",
+    outputType: 'blocks',
+    output: `enum SpriteKind {
+        Player,
+        Projectile,
+        Food,
+        Enemy
+    }
+    
+    let $spriteName = sprites.create(img\`
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    \`, SpriteKind.Player)`,
+    questions: [
+        {
+            "title": "What should your sprite be called?",
+            "inputs": [{
+                    "answerToken": "spriteName",
+                    "defaultAnswer": "mySprite",
+                    "type": "text"
+            }],
+            "output": "",
+            "goTo": 2
```

Comment:
nitpick (feel free to ignore nitpick comments): typically goto is lowercase, I'd just use "goto"

Reply:
Go to has been changed to lowercase. I've also standardized the goto object to take params.